VideoLocker/src/main/java/org/edx/mobile/third_party/iconify/IconDrawable.java

```diff
@@ -1,47 +1,40 @@
 package org.edx.mobile.third_party.iconify;
 
-/**
```

Comment:
Looks like the licence headers have been removed on the source repository as well, following a rewrite: JoanZapata/android-iconify@b08f28a10d050beaba6250e9e9c46efe13d9caaa

I just copied the file over from JoanZapata/android-iconify#134, but I guess we should probably retain the license.